packages/graphiql-plugin-utils/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./base.tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": false,
```

Comment:
```suggestion
    "allowSyntheticDefaultImports": true,
```

Reply:
this is not needed, as base config already has `true`. let's avoid repetitive config overrides